README.md

```diff
@@ -57,114 +59,97 @@ jobs:
       # Checks out your repository under $GITHUB_WORKSPACE, so that your job can access it.
       - uses: actions/checkout@v2
 
-      # The name for your step to display on GitHub.
-      - name: Run dotTEST analyzer
-        # Specifies the action to run as part of this step.
-        uses: parasoft/run-dottest-analyzer@1.0.0
+      # Runs code analysis with dotTEST.
+      - name: Run dotTEST
+        # Specifies the action to run.
+        uses: parasoft/run-dottest-action@1.0.0
```

Comment:
Change to: parasoft/run-dottest-action@1.1.0

Reply:
Done.